cmd/cmd.go

```diff
@@ -137,6 +138,15 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 			}
 
 			bar.Set(resp.Completed)
+		} else if resp.Quantize != "" {
```

Comment:
similar to above comment, could we just check which status was provided to choose how to render the status?